alchemy/src/cloudflare/certificate-pack.ts

```diff
@@ -314,96 +319,7 @@ export const CertificatePack = Resource(
     }
 
     if (this.phase === "update" && this.output?.id) {
-      // Validate immutable properties
-      const currentPack = this.output;
-
-      if (props.certificateAuthority !== currentPack.certificateAuthority) {
-        throw new Error(
-          `Cannot change certificateAuthority from '${currentPack.certificateAuthority}' to '${props.certificateAuthority}'. Certificate Authority is immutable after creation. You must delete and recreate the certificate pack.`,
-        );
-      }
-
-      if (
-        JSON.stringify(props.hosts.sort()) !==
-        JSON.stringify(currentPack.hosts.sort())
-      ) {
-        throw new Error(
-          `Cannot change hosts from [${currentPack.hosts.join(", ")}] to [${props.hosts.join(", ")}]. Hosts are immutable after creation. You must delete and recreate the certificate pack.`,
-        );
-      }
-
-      if (props.validationMethod !== currentPack.validationMethod) {
-        throw new Error(
-          `Cannot change validationMethod from '${currentPack.validationMethod}' to '${props.validationMethod}'. Validation method is immutable after creation. You must delete and recreate the certificate pack.`,
-        );
-      }
-
-      if (props.validityDays !== currentPack.validityDays) {
-        throw new Error(
-          `Cannot change validityDays from ${currentPack.validityDays} to ${props.validityDays}. Validity period is immutable after creation. You must delete and recreate the certificate pack.`,
-        );
-      }
-
-      const type = props.type || "advanced";
-      if (type !== currentPack.type) {
-        throw new Error(
-          `Cannot change type from '${currentPack.type}' to '${type}'. Type is immutable after creation. You must delete and recreate the certificate pack.`,
-        );
-      }
-
-      // Only cloudflareBranding can be updated
-      if (props.cloudflareBranding !== currentPack.cloudflareBranding) {
```

Comment:
cloudflare branding adds a cloudflare url to the certificate, so you can't actually update it despite what the api lets you do